$NetBSD: patch-Makefile,v 1.1 2024/05/11 15:34:57 wiz Exp $

Always link against libcurses and let the pkgsrc framework decide
which library it provides.

--- Makefile.orig	2020-12-20 18:01:30.000000000 +0000
+++ Makefile
@@ -13,7 +13,7 @@ MANPREFIX = ${PREFIX}/share/man
 # use system flags.
 CATPOINT_CFLAGS = ${CFLAGS}
 CATPOINT_CPPFLAGS = ${CPPFLAGS} -D_DEFAULT_SOURCE
-CATPOINT_LDFLAGS = ${LDFLAGS} -lncursesw
+CATPOINT_LDFLAGS = ${LDFLAGS} -lcurses
 # Gentoo
 #CATPOINT_LDFLAGS = ${LDFLAGS} -lncursesw -ltinfow
 
